governance/handler.go

```diff
@@ -127,20 +173,43 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
+	if GovernanceItems[k].checkValueType(uint64(0)) && reflect.TypeOf(val) == float64T {
 		x = uint64(val.(float64))
 		if float64(x.(uint64)) == val.(float64) {
 			return x
 		}
 		return val
 	}
 
+	if (k == params.AddValidator || k == params.RemoveValidator) && reflect.TypeOf(val) == stringT {
+		addresses := strings.Split(val.(string), ",")
+
+		if len(addresses) == 0 {
+			return val
+		}
+
+		if len(addresses) == 1 {
+			x = common.HexToAddress(val.(string))
+			return x
+		}
+
+		var nodeAddresses []common.Address
+		for _, str := range strings.Split(val.(string), ",") {
+			if common.IsHexAddress(str) {
+				nodeAddresses = append(nodeAddresses, common.HexToAddress(str))
+			} else {
+				return val
+			}
+		}
+		x = nodeAddresses
+		return x
+	}
+
 	// address comes as a form of string from JS console
-	if reqType == addressT && reflect.TypeOf(val) == stringT {
+	if GovernanceItems[k].checkValueType(common.Address{}) && reflect.TypeOf(val) == stringT {
```

Comment:
@yoomee1313 
Is it ok to pass `common.Address{}` to checkValueType, not val from caller?

Reply:
As I wrote at above comment, the condition checks the `reqType` of key `k`.

Reply:
@yoomee1313 

Understood. Thanks :)